http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutesSpec.scala

```diff
@@ -69,21 +69,16 @@ class ProxyRoutesSpec
 
   val prefix = "proxy"
 
-  "ProxyRoutes" should s"listen on /$prefix/{project}/{name}/... ($prefix)" in {
-    Get(s"/$prefix/$googleProject/$clusterName").addHeader(Cookie(tokenCookie)) ~> proxyRoutes.route ~> check {
-      handled shouldBe true
-      status shouldEqual StatusCodes.OK
-      validateCors()
-    }
-    Get(s"/$prefix/$googleProject/$clusterName/foo").addHeader(Cookie(tokenCookie)) ~> proxyRoutes.route ~> check {
+  "ProxyRoutes" should s"listen on /$prefix/{project}/{name}/jupyter)" in {
```

Comment:
So, I changed the proxy route so `/project/name/x` won't match to jupyter, and we instead use 'x' to determine if it should go to jupyter/rstudio (existing proxy service) or if its a service that should go to kubernetes. These tests aren't working, because the proxy route path matching is wrong currently